docs/src/reference/mount.md

```diff
@@ -9,968 +9,856 @@ status: enterprise
 !!! info
     Available in **lakeFS Cloud** and **lakeFS Enterprise**
 
-Everest is a complementary binary to lakeFS that allows users to virtually mount a remote lakeFS repository onto a local directory.
-Once mounted, users can access the data as if it resides on their local filesystem, using any tool, library, or framework that reads from a local filesystem.
+Everest is a complementary binary to lakeFS that allows you to virtually mount a remote lakeFS repository onto a local directory or within a Kubernetes environment. Once mounted, you can access data as if it resides on your local filesystem, using any tool, library, or framework.
 
-!!! note
-    No installation is required. Please [contact us](http://info.lakefs.io/thanks-lakefs-mounts) to get access to the Everest binary.
-
-!!! tip
-    Everest mount supports writing to the file system for both NFS and FUSE protocols starting version **0.2.0**!
-    
-    [Everest mount write mode semantics →](#mount-write-mode-file-system-behavior).
-
-
-<iframe width="420" height="315" src="https://www.youtube.com/embed/BgKuoa8LAaU"></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/BgKuoa8LAaU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## Use Cases
 
-* **Simplified Data Loading**: With lakeFS Mount, there's no need to write custom data loaders or use special SDKs. You can use your existing tools to read and write files directly from the filesystem.
-* **Handle Large-scale Data Without changing Work Habits**: Seamlessly scale from a few local files to millions without changing your tools or workflow. Use the same code from early experimentation all the way to production.
-* **Enhanced Data Loading Efficiency**: lakeFS Mount supports billions of files and offers fast data fetching, making it ideal for optimizing GPU utilization and other performance-sensitive tasks.
-
-## Requirements
-
-- For enterprise installations: lakeFS Version `1.25.0` or higher.
-
-### OS and Protocol Support
+- **Simplified Data Loading**: Use your existing tools to read and write files directly from the filesystem with no need for custom data loaders or SDKs.
+- **Seamless Scalability**: Scale from a few local files to billions without changing your tools or workflow. Use the same code from experimentation to production.
+- **Enhanced Performance**: Everest supports billions of files and offers fast, lazy data fetching, making it ideal for optimizing GPU utilization and other performance-sensitive tasks.
```

Comment:
The use case title sounds like a technical benefit of mount. But there is a use case for performant data loading which I believe should be highlighted instead. WDYT?